client/js/genre.js

```diff
@@ -1,14 +1,32 @@
-/** fetches youtube api calls from YoutubeServlet.java
-    and displays on youtube-genre.html
-*/
+/**
+ * fetches youtube api calls from YoutubeServlet.java
+ * and displays on youtube-genre.html
+ */
 
 const genreBlock = document.getElementById('genres');
+const numVideosInput = document.getElementById('numVideos');
+const DEFAULT_NUM_VIDS = 10;
+
+/**
+ * extracts num vids from input, otherwise returns default
+ * helper fn for displayMusicGenre()
+ * @returns {number} num videos user wants to retrieve
+ */
+function getNumVids() {
+  const numVideos = numVideosInput.value;
+  return numVideos || DEFAULT_NUM_VIDS;
```

Comment:
nit: just return `numVideosInput.value || DEFAULT_NUM_VIDS`. Creating an intermediary const doesn't improve readability at all so it's not necessary.